src/handler/index.ts

```diff
@@ -53,17 +56,28 @@ export const handler = async (event: APIGatewayProxyEvent, context: Context): Pr
     })
     .catch((error) => {
       const errorString: string = JSON.stringify(error, Object.getOwnPropertyNames(error));
-      log.error(`An unexpected error occured when fetching ATFs: ${errorString}`);
+      log.error(`An unexpected error occurred when fetching ATFs: ${errorString}`);
       throw error;
     });
+  // Filter out those with no availability
+  if (removeNoAvailabilityFlag) {
+    try {
+      atfs.Items = filterAtfs.removeAtfsWithNoAvailability(atfs.Items);
+      log.info(`ATFs after filtering out those with no availability [${atfs.Count}]`);
+    } catch (error) {
+      const errorString: string = JSON.stringify(error, Object.getOwnPropertyNames(error));
+      log.error(`An unexpected error occurred when filtering ATFs with no availability: ${errorString}`);
+      throw error;
```

Comment:
Do you know what circumstances would cause it to throw? Instead of throwing the error, can we skip the item that error'd or simply return the un-filtered list of atfs? So the user will still be able to see atfs and the error will be logged for us.

Reply:
Yeah there shouldn't be a way for it to fail actually. Refactored